K8s/easy-shop-migration-job.yml

```
@@ -0,0 +1,19 @@
+    environment: production
+spec:
+  template:
+    spec:
+      containers:
+      - name: migration
+        image: devshubh2204/easyshop-hack_migration:latest
+        imagePullPolicy: Always
+        env:
+        - name: MONGODB_URI
+          value: "mongodb://mongodb-service:27017/easyshop"
+      restartPolicy: OnFailure
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)